epoxy-processor/src/main/java/com/airbnb/epoxy/processor/ModelBuilderInterfaceWriter.kt

```diff
@@ -50,7 +50,12 @@ class ModelBuilderInterfaceWriter(
             val interfaceMethods = getInterfaceMethods(modelInfo, methods, interfaceName)
 
             if (modelInfo is ModelViewInfo) {
-                modelInfo.generatedViewInterfaceNames.forEach { viewInterface ->
```

Comment:
can we delete the property `generatedViewInterfaceNames` now?

Reply:
makes sense!